frameworks/g-cloud-9/questions/services/standardsCSASTARLevel.yml

```diff
@@ -0,0 +1,26 @@
+name: CSA STAR certification level
+question: What level is the certification?
+
+hidden: true
+depends:
+  - "on": lot
+    being:
+      - cloud-hosting
+      - cloud-software
+
+type: radios
+options:
+  - label: "Level 1: CSA STAR Self-Assessment"
+    value: level_1
```

Comment:
More detail in the values here might not hurt? e.g. `level_1_csa_star_attestation`?  But I'm not strong on this.